#BodySize.msg

Header header
int32 id
float32 head_feet_distance